test-cypress/fixtures/locale-en.json

```diff
@@ -963,6 +963,8 @@
     "repoTooltips.throwQueryEvaluationExceptionOnTimeout": "Repository throws QueryEvaluationException when the duration of a query execution exceeds the timeout parameter.",
     "repoTooltips.queryLimitResults": "Sets the maximum number of results returned from a query after which the evaluation of a query will be terminated; values less than or equal to zero mean no limit.",
     "repoTooltips.nonInterpretablePredicates": "Semicolon-delimited list of predicates (full URIs) that GraphDB will not try to process with the registered GraphDB plugins.",
+    "repoTooltips.fts": "Enable full-text search.",
+    "repoTooltips.ftsLanguages": "Sets languages and respectively the appropriate analyzers for enabled full-text search.",
```

Comment:
White space-delimited list of language that should have specific index with appropriate analyzer for full-text search.